run-17495340446-artifacts/scripts/audit_style.py

```
@@ -0,0 +1,389 @@
+                    })
+    
+    return issues
+
+def audit_banned_words(rows: List[Tuple]) -> List[Dict[str, Any]]:
+    """Check for banned phrases that indicate tone drift."""
+    issues = []
+    
+    banned_pattern = '|'.join(r'\b' + re.escape(phrase) + r'\b' for phrase in BANNED_PHRASES.keys())
+    regex = re.compile(banned_pattern, re.IGNORECASE)
+    
+    for slide_xml, idx, jp, en, kind in rows:
```

Ruff (0.12.2)

Loop control variable `jp` not used within loop body

Rename unused `jp` to `_jp`

(B007)